bananas_api/new_upload/classifiers/scenario.py

```diff
@@ -0,0 +1,69 @@
+from ...helpers.api_schema import Classification
+from ...helpers.enums import (
+    Climate,
+    Resolution,
```

Comment:
I know heightmaps use "Resolution", but for scenarios I would prefer "size: small/large/huge".

For heightmaps it makes sense to have large resolutions, since you can always generate a smaller map. For single-player a 512x1024 map may not be "normal" sized.